plugins/modules/rds_instance.py

```diff
@@ -334,7 +334,7 @@
         description:
           - An integer that specifies the order in which an Aurora Replica is promoted to the primary instance after a failure of
             the existing primary instance.
-        type: str
```

Comment:
nominally this would be a "breaking" change, however passing anything here would currently result in an exception, so I think we let this pass.